samples/variables_and_state/variables_and_state.ipynb

```diff
@@ -477,12 +477,12 @@
               "<IPython.core.display.Javascript object>"
             ],
             "application/javascript": [
-              "download(\"download_1285be07-105d-4d50-84e7-523370341c4a\", \"variables_and_state_colab_artifacts.zip\", 4286)"
+              "download(\"download_4d24e1f7-a107-4c83-967d-7f9267f88e02\", \"variables_and_state_colab_artifacts.zip\", 4368)"
             ]
           },
           "metadata": {}
         }
       ]
     }
   ]
-}
+}
```

Comment:
No newline :angry: (don't manually edit though)

Reply:
lol yeah, if you download from colab.research.google.com the newlines aren't included
Anytime someone makes manual edits, their editor usually puts them back